mapit_global/templates/mapit/description.html

```diff
@@ -0,0 +1 @@
+Points-to-boundary mapping, anywhere in the world. Input a zipcode or geographical location, and MapIt returns the relevant constituency or authority.
```

Comment:
There aren't any zipcodes in the global installation; 'administrative area' instead of 'constituency or authority', I think.